src/entity/View.ts

```diff
@@ -21,7 +21,7 @@ export class View {
   @Column({ type: 'text', nullable: true })
   referer?: string;
 
-  @PrimaryColumn({ default: () => 'now()', precision: 3 })
+  @PrimaryColumn({ default: () => 'now()', precision: 0 })
```

Comment:
I prefer to not introduce any change to existing columns in this table as it contains a lot of data.
Unless it's mandatory

Reply:
Understood, with what, whenever we query data for comparison of timestamp, we will have to strongly type the actual query every time even for basic usage so we can parse it by removing the milliseconds and having it matched. I will update the current mutation.

I just felt the process in this PR can benefit future developments as we won't have to bother to do the parsing from the DB query, but yes, it is much better to avoid changes in the structure that will affect the data itself.

Reply:
We also decided to not compare timestamps at the moment so this shouldn't be a problem